syncmarkets/clients/coingecko/cache.go

```diff
@@ -92,3 +92,12 @@ func getCoinsMap(coins GeckoCoins) map[string]GeckoCoin {
 	}
 	return coinsMap
 }
+
+func normalizeTokenId(platform, addr string) string {
```

Comment:
can you create unit tests for that?

Reply:
https://github.com/trustwallet/blockatlas/pull/806/commits/861dd8f8a4e501d120e3f45395623930b88a6e63